utils/doclint/generateApiJson.js

```diff
@@ -59,19 +58,28 @@ function serializeMember(member) {
 
 function serializeProperty(arg) {
   const result = { ...arg };
-  delete result.args;
-  delete result.argsArray;
+  sanitize(result);
   if (arg.type)
     result.type = serializeType(arg.type)
   return result;
 }
 
+function sanitize(result) {
+  delete result.args;
+  delete result.argsArray;
+  delete result.templates;
+  if (result.properties && !Object.keys(result.properties).length)
+    delete result.properties;
+}
+
 function serializeType(type) {
   const result = { ...type };
-  if (type.properties) {
+  if (type.properties && type.properties.length) {
```

Comment:
we can also save some bytes by removing other empty/default value fields (required, returnComment etc)

Reply:
It does not save much...